### DISCUSSION TITLE
changing language

### DISCUSSION THREAD

**Discussion by bryant-the-coder** [i3/i3#5218] [Q&A]
how to change language from english to chinese in i3?

**Comment by Airblader** [comment]
Language of what?

**Reply by bryant-the-coder** [reply]
so by default i m using english (UK), i also use chinese to communicate. how can i change the language for the keyboard?

**Reply by Airblader** [reply]
https://wiki.archlinux.org/title/Xorg/Keyboard_configuration#Setting_keyboard_layout

Using setxkbmap is pretty much the default way to go.
